### DISCUSSION TITLE
yocto dunfell

### DISCUSSION THREAD

**Discussion by ZahraAlidoost** [community/community#67168] [Programming Help] [answered]
https://github.com/megous/linux.git not found

Question

### Body

hello
in yocto3.1 i want orange-pi-5.8, in linux-mainline_5.8.16.bb there is https://github.com/megous/linux.git;protocol=git;branch=orange-pi-5.8 but it is  Not Found

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ZahraAlidoost ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by aliasghar1379** [comment, marked as answer]
Hi same problem here and searching about it to fix it and still nothing for sure ......
